blog/2023-07-3.4-3.13.md

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Uppy 3.3 to 3.13: conditional S3 multipart, signing on the client, speedy handling of 10k files and much much more'
+date: 2023-07-26
+authors: [murderlon, arturi, aduh95, mifi]
+image: '/img/blog/3.4-3.13/single-file-mode.jpg'
+slug: '2023/07/3.3-3.13'
+published: true
+---
+
+<img src="/img/blog/3.4-3.13/single-file-mode.jpg" className="border" alt="Screenshot of the new Uppy website homepage" />
+
+This is a big Uppy update post, covering relworkeases from [`3.3.0` to `3.13.0`](https://github.com/transloadit/uppy/compare/uppy%403.4.0...uppy%403.13.0)! 
+
+In this issue: the long-awaited conditional S3 Multipart, much improved performance when adding and validating 10k+ files, Server-sent events, assemblyOptions for Transloadit, stability improvements and bug fixes. 
+
+Please make a cup of something tasty ☕️ (in a non-spillable container), as this will be quite a ride.
+
+<!--truncate-->
+
+## Amazon S3
```

Comment:
I think we also want a specific section to address the deprecation of `@uppy/aws-s3-multipart` and the legacy `@uppy/aws-s3`.

```suggestion
## Amazon S3

### Merging of the plugins

After years of separating regular non-multipart uploads and multipart uploads, we are finally reuniting the two sibling plugins. As of now:

- `@uppy/aws-s3-multipart` is deprecated. You should use `@uppy/aws-s3` and set the `shouldUseMultipart: true` option to get the same behavior. We plan to keep it as an alias for the next major cycle and add a warning at the install when Uppy 4.0.0 will be released.
- `@uppy/aws-s3` now hosts two plugins:
  - the legacy one, untouched, to guarantee backward compatibility. We plan to get rid of it in the next major, we recommend moving away from it and report if you see something missing in the new plugin.
  - the new merged plugin, available if you pass `shouldUseMultipart` option.

Our plan for this merge is to maintain the backward compatibility until the next major, while also providing forward compatibility so you can try our new plugin and give us some feedback before we remove the old plugins for good.
```